README.md

```diff
@@ -0,0 +1,14 @@
+Run in project root directory:
+
+```shell
+flatpak remote-add --if-not-exists --user flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+flatpak install --user -y flathub org.flatpak.Builder
+python -m pip install '.[gui,basicvsrpp,packaging-flatpak]'
+pip-compile --extra gui,basicvsrpp -o flatpak/requirements.txt setup.py
+req2flatpak --requirements-file flatpak/requirements.txt --target-platforms 312-x86_64 > flatpak/lada-pip-dependencies.json.tmp
```

Comment:
so what happen on aarch64?

Reply:
For now I don't target to support ARM as I cannot test it myself. I currently also assume the user to have a CUDA capable / Nvidia GPU.

Reply:
This is not what the build think. It will try to build an aarch64

Also assuming nvidia is very wrong. What happen if the user doesn't have?

Reply:
At least one of my core dependencies [MMC](https://mmcv.readthedocs.io/en/latest/get_started/installation.html#install-with-pip) only mentions CUDA GPU support. Pytorch seems to try to support more AMD and Intel GPUs recently but the ecosystem as a whole still seems to assume CUDA only.

I could try to export the models to ONNX in a later version as it has more compatibility across systems.

Reply:
this doesn't tell what happens.  Does it crash? Does it just do nothing? Does it tell the user?

Reply:
When no GPU could be detected the app still technically works and will use the CPU (tested on x86_64) but I've added a banner to indicate that it's probably too slow to be useful.

![screenshot_no_gpu_banner](https://github.com/user-attachments/assets/215f0d2c-1884-4e0f-9050-8d1e9c76be35)

Regarding aarch64: I wasn't successful trying to install pip requirements for aarch64. I'm not sure how the app would behave if started on aarch64 right now. Guess I have to look into aarch64 on x86_64 emulation so I can test this...

Reply:
The following dependencies do not provide aarch64 wheels
```
nvidia-cublas-cu12
nvidia-cuda-cupti-cu12
nvidia-cuda-nvrtc-cu12
nvidia-cuda-runtime-cu12
nvidia-cudnn-cu12
nvidia-cufft-cu12
nvidia-curand-cu12
nvidia-cusolver-cu12
nvidia-cusparse-cu12
nvidia-nvtx-cu12
triton
mmcv
```
For nvidia there seems to be options directly from nvidia but not provided by pytorch as far as I can tell
Triton (dependency of pytorch) is not available yet on aarch64: [GH issue](https://github.com/pytorch/pytorch/issues/130558)

I tried to skip them as they are technically not necessary and torch should fall back to CPU inference on aarch64

The other dependency mmcv is necessary but they only provide x86 builds. aarch64 seems to be possible but would have to built it from source

[GH discussion](https://github.com/open-mmlab/mmcv/discussions/2334)
[source build docs](https://mmcv.readthedocs.io/en/latest/get_started/build.html)

Setting up cross compilation builds and integrating this into flatpak manifest source creation via req2flatpak is over my head.

**TLDR;** If aarch64 support is a pre-requisite for submission to flathub I'd need to withdraw my submission for this app

Please let me know, the I'll close the PR

Reply:
The prerequisite it to not build broken packages. If you package doesn't work on aarch64, it current is still build on that platform.

Reply:
Thanks for the hint. Just found out about `flathub.json` and that I can specify which archs to build for